Compiler/BackEnd/VisualXML.mo

```diff
@@ -169,19 +169,31 @@ author: vwaurich 2016-10"
   input BackendDAE.Variables vars;
   output DAE.Exp eOut;
 protected
+  String s;
   DAE.Exp e;
   BackendDAE.Var var;
 algorithm
   try
     ({var},_) := BackendVariable.getVar(cr,vars);
     e := BackendVariable.varBindExp(var);
-    if Expression.isConst(e) then
-      eOut := e;
-    elseif Expression.isCref(e) then
-      eOut := getConstCrefBinding(Expression.expCref(e),vars);
-    else
-      Error.addInternalError("VisualXMl.getConstCrefBinding failed for "+ExpressionDump.printExpStr(e)+"\n", sourceInfo());
```

Comment:
Easier would be to just give eOut a value. You now also have a try-else block that can't be reached.